src/data/int/basic.lean

```diff
@@ -128,7 +131,7 @@ theorem nat_abs_neg_of_nat (n : ℕ) : nat_abs (neg_of_nat n) = n :=
 by cases n; refl
 
 theorem nat_abs_mul (a b : ℤ) : nat_abs (a * b) = (nat_abs a) * (nat_abs b) :=
-by cases a; cases b; simp [(*), int.mul, nat_abs_neg_of_nat]
+by cases a; cases b; simp only [has_mul.mul, int.mul, nat_abs_neg_of_nat, eq_self_iff_true, int.nat_abs]
```

Comment:
The `only` flag is important but the change from `(*)` to `has_mul.mul` is not necessary.

Reply:
Reverted, oops.